cartoframes/data/observatory/geography.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
+
+from cartoframes.data.observatory.repository.dataset_repo import get_dataset_repo
+from cartoframes.data.observatory.repository.geography_repo import get_geography_repo
+
+_GEOGRAPHY_FIELD_ID = 'id'
+
+
+class Geography(pd.Series):
```

Comment:
I'm seeing in the data model, that `Geography` has a `geom_coverage` column of type `GEOMETRY`. How are we dealing with this type of column?

Reply:
We're not, panda is dealing with it. We don't deal with columns anymore. Does it sound right?

Reply:
👍 It's not important right now, but let's take into account in the future, since if we want to do some spatial operation, we'd need to map Geography as a geodataframe instead of a dataframe.